.github/ISSUE_TEMPLATE/NEW_RELEASE.md

```diff
@@ -50,12 +51,17 @@ Please do not remove items from the checklist
 - [ ] For major or minor releases, merge the `main` branch into the `website` branch to publish the updated documentation.
 - [ ] Send an announcement email to `sig-scheduling@kubernetes.io` and `wg-batch@kubernetes.io` with the subject `[ANNOUNCE] kueue $VERSION is released`.   <!--Link: example https://groups.google.com/a/kubernetes.io/g/wg-batch/c/-gZOrSnwDV4 -->
 - [ ] Update the below files with respective values in `main` branch :
-  - Latest version in `README.md`
-  - Latest version in `cmd/experimental/kjobctl/docs/installation.md`
+  - Latest version in:
+    - `README.md`
+    - `Makefile`
+    - `installation.md`
+    - `cmd/experimental/kjobctl/docs/installation.md`
   - Release notes in the `CHANGELOG`
   - `version` in `site/hugo.toml`
   - `appVersion` in `charts/kueue/Chart.yaml`
   - `SECURITY-INSIGHTS.yaml` values by running `make update-security-insights GIT_TAG=$VERSION`
+  - Submit a pull request with the changes: <!-- example #3007 -->
+  - Cherry-pick the pull request onto the release branch
```

Comment:
Did you mean `website` branch?

Reply:
Yeah, I meant the analog of https://github.com/kubernetes-sigs/kueue/pull/2556